development-workflow.md

```diff
@@ -0,0 +1,61 @@
+# MeiliSearch Development Workflow
+
+MeiliSearch is an open soure project. 
+
+This document provides a general overview of the workflow for its development. 
+
+This document covers only the workflow for the [main project](meilisearch/MeiliSearch).
+
+[TOC]
+
+## Governance
+
+### MeiliSearch and Meili (the company)
+MeiliSearch is supported and developed mostly by Meili, the company. The project aims to be more and more managed by the community at large. Teams membership are, in theory, open to anyone working, or not, for Meili, the company.
+
+> __Disclaimer__
+> _The description of the teams and governance in this document are completely unrelated to how the company works and is organized internally._
+
+## 🔮 Roadmap
+The MeiliSearch roadmap is public on [roadmap.meilisearch.com](https://roadmap.meilisearch.com).
+Anyone can vote for future features or submit an new idea. The roadmap is moderated and prioritized by MeiliSearch CEO.
+When an idea on the roadmap is prioritized it will enter in the development workflow.
+
+> __Note__
+> Not all specifications must be born out of a card on the roadmap and not all roadmap cards will be transformed into specifications.
+
+
+## 🎼 Development Workflow
+Many changes including bug fixes or documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow which won't be covered in this document.
+
+Some changes are more substantial and we ask these to be put through a design process and produce consensus among the MeiliSearch community  and teams. 
+As a rule of thumb, any changes that impact in any way the user of MeiliSearch (without it being a bugfix) should follow this workflow.
+
+## 📝 Specifications (or RFCs) process
```

Comment:
This one and the next H@ should be H3 since they are part of the development process, unless I misunderstood?

Reply:
Updated in next commit